source/exe/main_common.cc

```diff
@@ -50,6 +53,9 @@ MainCommonBase::MainCommonBase(const OptionsImpl& options, Event::TimeSystem& ti
                                Filesystem::Instance& file_system)
     : options_(options), component_factory_(component_factory), thread_factory_(thread_factory),
       file_system_(file_system), stats_allocator_(symbol_table_) {
+#ifdef ENVOY_GOOGLE_GRPC
+  grpc_init();
```

Comment:
Ahhh this is where I got stuck. For some reason I though one of the c_smt_ptrs was handling this.